integration-tests/templates/rn71-template/App.tsx

```diff
@@ -7,7 +7,7 @@ function App(): React.JSX.Element {
     <EmbraceTestHarness
       navigationStyle="react-native"
       sdkConfig={sdkConfig}
-      allowCustomExport={true}
+      allowCustomExport={false}
```

Comment:
@facostaembrace I originally switched this to false for testing because the package wasn't available for Android, however even though that's available now I'm thinking it may make sense to keep it set as false since that will be the more common code path our users take, what do you think?

Reply:
I think I agree on that, probably I can try to run this suite locally following the steps you mentioned earlier to confirm that the new otlp works fine for all rn versions before I release the component. that should be enough for testing purposes. but I agree that we don't need this turned on for every release

Reply:
another option maybe would be to commit an app to the repo that is configured for custom export and then have certain tests that only run with that one specifically

or actually maybe simpler would be to add some button to the test harness that restarts the app and flips it to use custom export